### ISSUE TITLE
limit # of mentions in a single post

### ISSUE THREAD

**Issue by barisusakli** [julianlam/nodebb-plugin-mentions#65]
maybe no more than 5 allowed

https://github.com/NodeBB/NodeBB/issues/4395

**Comment by asdf-tdwtf** [comment]
Please make sure you make this configurable if you implement it. If you reply to multiple posts at once or play forum games, you might want to be able to mention more than 5 people at the same time.

Also, I don't the mentions are the actual root cause of the bug I reported. It seemed more like a bug with frontend rendering than the mentions themselves.

**Comment by julianlam** [comment]
Yeah, I'm not entirely sure this should be "resolved" in mentions, only since the rendering issue was due to the browser (maybe?) flipping it's tables.

Though in the above case with mafia, you'd probably want to put all of those users in a user group so they can be mentioned via a single tag.
